tests/rest/RestBase.php

```diff
@@ -198,7 +197,7 @@ protected function getCategory() {
 	 * @return string
 	 */
 	protected function getTestCaseReference() {
-		return static::class . '::' . $this->getName();
+		return $this->toString();
```

Comment:
That actually makes our own getTestCaseReference() method useless and redundant, might as well get rid of it completely and replace it by toString() wherever it's used...

Reply:
You are quite right, indeed we can simplify the code and it will even become similar to the ‘soap’ test suite.